docs/_newsfragments/1589.breakingchange.rst

```diff
@@ -0,0 +1,3 @@
+Deserializing an empty JSON body on a request will return a BadRequest.
```

Comment:
Should we probably be more specific that this deserialization pertains to `req.media` / `req.get_media()`?